numpy/core/tests/test_multiarray.py

```diff
@@ -2416,12 +2416,41 @@ def test_sort_unicode_kind(self):
         assert_raises(ValueError, d.argsort, kind=k)
 
     def test_searchsorted(self):
```

Comment:
Would be good to parameterize this test.

Reply:
I parameterized the tests for real floats. I also split up the rest of the function into smaller tests. Not the best, but perhaps more readable.